layouts/partials/carousel/modal-content.html

```diff
@@ -7,7 +7,7 @@
     '</div>' +
     '<div class="tvp-products-holder">' +
       '<div class="tvp-products"></div>' +
-      '<div class="tvp-arrow-indicator"></div>' +
+      '<div class="tvp-arrow-indicator"><div class="tvp-inner-arrow-indicator"></div></div>' +
```

Comment:
Is this driven by a design?

Reply:
this was a fix for the modal `popUP` arrow in general, it wasn't displaying correctly

Reply:
What was the issue?

Reply:
styling the arrow was not consistent, didn't look very good when changing styles

Reply:
:+1: